pyang/statements.py

```diff
@@ -2118,6 +2118,47 @@ def v_reference_deviation(ctx, stmt):
     stmt.i_target_node = find_target_node(ctx, stmt)
 
 def v_reference_deviate(ctx, stmt):
+    def search_ancestor_config_false(node, target):
```

Comment:
I wonder if this is correct / necessary.  I think it should be sufficient to check parent.i_config.  Did you consider that?

Reply:
Yes, It's a good point. I will try it tomorrow.